app/routes/events/components/EventAttendeeStatistics.js

```diff
@@ -0,0 +1,283 @@
+// @flow
+
+import { DistributionDataPoint } from 'app/components/Chart/utils';
+import type { Dateish, EventRegistration } from 'app/models';
+import DistributionPieChart from 'app/components/Chart/PieChart';
+import ChartLabel from 'app/components/Chart/ChartLabel';
+import { Flex } from 'app/components/Layout';
+import {
+  CartesianGrid,
+  Legend,
+  Line,
+  LineChart,
+  Tooltip,
+  XAxis,
+  YAxis,
+} from 'recharts';
+import moment from 'moment';
+
+interface RegistrationDateDataPoint {
+  name: string;
+  registrations: number;
+  unregistrations: number;
+}
+
+const ABAKOM_GROUP_IDS = [4, 5, 6, 7, 8, 9, 10, 11, 98, 231];
+const REVY_GROUP_IDS = [59, 119, 120, 121, 123, 124, 153, 154, 214, 222, 225];
+
+const PieChartWithLabel = ({
+  label,
+  distributionData,
+}: {
+  label: string,
+  distributionData: DistributionDataPoint[],
+}) => {
+  return (
+    <>
+      <h4>{label}</h4>
+      <Flex alignItems="center">
+        <DistributionPieChart
+          dataKey="count"
+          distributionData={distributionData}
+        />
+        <ChartLabel distributionData={distributionData} />
+      </Flex>
+    </>
+  );
+};
+
+const toLocalizedGender = (gender: string) => {
+  if (gender === 'male') return 'Mann';
+  if (gender === 'female') return 'Kvinne';
+  return 'Annet';
+};
+
+const createGenericDataPoint = (
```

Comment:
Perhaps a better name for this might be either `addGenericDataPoint` or `addOrIncrementDataPoint`? I was a bit confused by the naming :grimacing:

Reply:
✅